pkg/p2p/libp2p/connections_test.go

```diff
@@ -604,7 +604,7 @@ func TestWithBlocklistStreams(t *testing.T) {
 	expectPeersEventually(t, s2)
 	expectPeersEventually(t, s1)
 
-	if _, err := s2.Connect(ctx, s1_underlay); err == nil {
+	if _, _, err := s2.Connect(ctx, s1_underlay); err == nil {
```

Comment:
i see that all tests here ignore the returned value from `Connect`. It might be good to have a test that defines:
- 1 light node
- 1 full node
- connect from full to light
- check that the bool is `false`

Reply:
added